### ISSUE TITLE
Marathon crash when syncing tasks

### ISSUE THREAD

**Issue by BrickXu** [mesosphere/marathon#1702]
Hi all, 

My marathon(0.8.1) crash today and I found some error logs, does it fix in the 0.8.2 ?

```
[2015-06-24 18:07:03,238] INFO Syncing tasks for all apps (mesosphere.marathon.SchedulerActions:445)
[2015-06-24 18:07:03,240] INFO Requesting task reconciliation with the Mesos master (mesosphere.marathon.SchedulerActions:472)
[2015-06-24 18:07:03,242] INFO Already running 1 instances of /logstash/logs/error. Not scaling. (mesosphere.marathon.SchedulerActions:554)
[2015-06-24 18:07:03,243] INFO Already running 5 instances of /logstash/logs/kernel. Not scaling. (mesosphere.marathon.SchedulerActions:554)
[2015-06-24 18:07:03,243] INFO Already running 1 instances of /dns. Not scaling. (mesosphere.marathon.SchedulerActions:554)
[2015-06-24 18:07:03,244] INFO Already running 12 instances of /logstash/logs/all. Not scaling. (mesosphere.marathon.SchedulerActions:554)
[2015-06-24 18:07:03,245] INFO Received status update for task docker_registry.5bda112c-1438-11e5-aa0e-fa163ee98acb: TASK_RUNNING (Reconciliation: Latest task state) (mesosphere.marathon.MarathonScheduler:149)
[2015-06-24 18:07:03,246] INFO Already running 1 instances of /docker/registry-ui. Not scaling. (mesosphere.marathon.SchedulerActions:554)
[2015-06-24 18:07:03,247] INFO Already running 1 instances of /docker/registry. Not scaling. (mesosphere.marathon.SchedulerActions:554)
[2015-06-24 18:07:03,248] INFO Already running 1 instances of /jenkins. Not scaling. (mesosphere.marathon.SchedulerActions:554)
[2015-06-24 18:07:03,249] INFO Already running 10 instances of /statsd. Not scaling. (mesosphere.marathon.SchedulerActions:554)
[2015-06-24 18:07:03,256] INFO Received status update for task logstash_logs_all.a8546394-18dd-11e5-8911-fa163e678c5d: TASK_RUNNING (Reconciliation: Latest task state) (mesosphere.marathon.MarathonScheduler:149)
F0624 18:07:03.264706  2754 check.hpp:79] Check failed: f.isReady()
*** Check failure stack trace: ***
    @     0x7f030a989d60  google::LogMessage::Fail()
    @     0x7f030a989cb9  google::LogMessage::SendToLog()
    @     0x7f030a989697  google::LogMessage::Flush()
    @     0x7f030a98c61f  google::LogMessageFatal::~LogMessageFatal()
    @     0x7f030a97c885  _checkReady<>()
    @     0x7f030a978640  Java_org_apache_mesos_state_AbstractState__1_1fetch_1get_1timeout
    @     0x7f03794eada2  (unknown)
```

**Comment by aquamatthias** [comment]
That is the state abstraction bug in mesos, that is fixed in mesos 0.22.1 and Marathon 0.8.2.

**Comment by BrickXu** [comment]
@aquamatthias  Thx.
